src/routes/messages/entities/message.entity.ts

```diff
@@ -0,0 +1,63 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { AddressInfo } from '../../common/entities/address-info.entity';
+import { MessageConfirmation } from './message-confirmation.entity';
+
+export enum MessageStatus {
+  NeedsConfirmation = 'NEEDS_CONFIRMATION',
+  Confirmed = 'CONFIRMED',
+}
+
+export class Message {
+  @ApiProperty()
+  messageHash: string;
+  @ApiProperty()
+  status: MessageStatus;
+  @ApiPropertyOptional()
```

Comment:
Don't we need to specify the nullable type in this annotation too?

Reply:
Yes, good catch, it's needed to make OpenAPI specification to show `nullable: true` flag. I've fixed it in [7f7e62e](https://github.com/5afe/safe-client-gateway-nest/pull/275/commits/7f7e62e3e71639a85bf9640f861913cb68f50e10)

We need to fix this as well for other routes ([example 1](https://github.com/5afe/safe-client-gateway-nest/blob/982d7b7392eae06d4b91f21bb82c40d0ba3651f4/src/routes/safes/entities/safe-info.entity.ts#L29), [example 2](https://github.com/5afe/safe-client-gateway-nest/blob/982d7b7392eae06d4b91f21bb82c40d0ba3651f4/src/routes/transactions/entities/settings-change-transaction.entity.ts#L9)). I will do it in a separate PR to avoid adding noise to this one 🙂